_src/pattern-library/sass/components/_grid.scss

```diff
@@ -0,0 +1,1125 @@
+    }
+
+    .lg-pre-10 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include pre(10); };
+    }
+
+    .lg-pre-11 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include pre(11); };
+    }
+
+    // shifts - post
+    .lg-post-1 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include post(1); };
+    }
+
+    .lg-post-2 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include post(2); };
+    }
+
+    .lg-post-3 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include post(3); };
+    }
+
+    .lg-post-4 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include post(4); };
+    }
+
+    .lg-post-5 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include post(5); };
+    }
+
+    .lg-post-6 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include post(6); };
+    }
+
+    .lg-post-7 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include post(7); };
+    }
+
+    .lg-post-8 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include post(8); };
+    }
+
+    .lg-post-9 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include post(9); };
+    }
+
+    .lg-post-10 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include post(10); };
+    }
+
+    .lg-post-11 {
+        @include susy-breakpoint($bp-screen-lg, $susy-manual) { @include post(11); };
+    }
+
+
+    // CASE: x-large screen bp
+    .xl-col-1 {
+        @include susy-breakpoint($bp-screen-xl, $susy-manual) { @include span(1); };
+    }
+
+    .xl-col-2 {
+        @include susy-breakpoint($bp-screen-xl, $susy-manual) { @include span(2); };
+    }
+
+    .xl-col-3 {
+        @include susy-breakpoint($bp-screen-xl, $susy-manual) { @include span(3); };
+    }
+
+    .xl-col-4 {
+        @include susy-breakpoint($bp-screen-xl, $susy-manual) { @include span(4); };
+    }
+
+    .xl-col-5 {
+        @include susy-breakpoint($bp-screen-xl, $susy-manual) { @include span(5); };
+    }
+
+    .xl-col-6 {
+        @include susy-breakpoint($bp-screen-xl, $susy-manual) { @include span(6); };
+    }
+
+    .xl-col-7 {
+        @include susy-breakpoint($bp-screen-xl, $susy-manual) { @include span(7); };
+    }
+
+    .xl-col-8 {
+        @include susy-breakpoint($bp-screen-xl, $susy-manual) { @include span(8); };
+    }
+
+    .xl-col-9 {
+        @include susy-breakpoint($bp-screen-xl, $susy-manual) { @include span(9); };
+    }
+
+    .xl-col-10 {
+        @include susy-breakpoint($bp-screen-xl, $susy-manual) { @include span(10); };
+    }
+
+    .xl-col-11 {
+        @include susy-breakpoint($bp-screen-xl, $susy-manual) { @include span(11); };
+    }
+
+    .xl-col-12 {
+        @include susy-breakpoint($bp-screen-xl, $susy-manual) { @include span(full); };
+    }
+
+    // shifts - pre
+    .xl-pre-1 {
```

Comment:
NOTE: This whole set of definitions could be made more efficient through a mixin or function, but that seemed like a clean up/optimization task once the grid gets some use.